ledger/src/blockstore_db.rs

```diff
@@ -135,6 +135,11 @@ pub enum IteratorMode<Index> {
 pub mod columns {
     #[derive(Debug)]
     /// The slot metadata column
+    /// This column family maintains the status of the received shred data of
+    /// each recent slot.  In high level, it helps validator to understand
+    /// whether a slot is ready to replay based on information such as whether
+    /// it has a parent slot and whether, whether the validator has received
+    /// all the shreds of the current slot, etc.
```

Comment:
nits:
- "maintains" ==> "tracks" , what do you think?
- "the received shred data of each recent slot" ==> "received shred data for a given slot"
- "In high level, it helps validator to understand whether a slot is ready to replay based on information such as whether it has a parent slot and whether, whether the validator has received all the shreds of the current slot, etc." ==> "Tracking the progress as the slot fills up allows us to know if the slot (or pieces of the slot) are ready to be replayed.